plugins/python-build/share/python-build/jython-2.7.0

```diff
@@ -0,0 +1,22 @@
+require_java
+install_jar "jython-2.7.0" "https://repo1.maven.org/maven2/org/python/jython-installer/2.7.0/jython-installer-2.7.0.jar#b44352ece72382268a60e2848741c96609a91d796bb9a9c6ebeff62f0c12c9cf" jython
+
+# Jython is supposed to install pip using ensurepip by default, but this does
+# not appear to be working so we do it manually.
+#
+# pyenv's `ensurepip` downloads `get-pip.py` which isnt currently compatible
+# with jython: http://bugs.jython.org/issue2302
+#
+# Use the ensurepip module in jython to install the bundled pip instead:
+"$PYTHON_BIN" -m ensurepip
```

Comment:
Jim Baker from the jython team informed me that the [jython installer jar should be run with the absolute path to the jar and destination directory](http://bugs.jython.org/msg10009). Fixed in e285f2e